dcos_cli/cli/cluster/__init__.py

```diff
@@ -1,14 +1,14 @@
-import click
-
 from dcos_cli.cli.dcos import dcos
 
+
 @dcos.group()
 def cluster():
     """Manage your DC/OS clusters."""
     pass
 
-import dcos_cli.cli.cluster.attach # noqa: E402
-import dcos_cli.cli.cluster.list # noqa: E402
-import dcos_cli.cli.cluster.remove # noqa: E402
-import dcos_cli.cli.cluster.rename # noqa: E402
-import dcos_cli.cli.cluster.setup # noqa: E402
+# pylint: disable=wrong-import-position
+import dcos_cli.cli.cluster.attach # noqa
```

Comment:
Why remove the specifics of the `noqa`? Using specific codes allows `flake8` to report issues that you didn't mean to ignore.

I don't understand why these are imported late. Please add a comment if there is a good reason, or move them to the top of the file.

Reply:
I was missing the noqa F401. As for these imports it's the same reason than here (https://github.com/mesosphere/dcos-security-cli/blob/master/dscli/__init__.py#L99).

I will check to see if there is another way to do this, that'd also allow to get rid of linter exceptions for them.

Reply:
I've switched to another way, cf. last commit.

Reply:
🙏